core/variant_call.cpp

```diff
@@ -1925,6 +1935,10 @@ void register_variant_methods() {
 
 	_VariantCall::add_constructor(_VariantCall::Color_init1, Variant::COLOR, "r", Variant::REAL, "g", Variant::REAL, "b", Variant::REAL, "a", Variant::REAL);
 	_VariantCall::add_constructor(_VariantCall::Color_init2, Variant::COLOR, "r", Variant::REAL, "g", Variant::REAL, "b", Variant::REAL);
+	//_VariantCall::add_constructor(_VariantCall::Color_init3, Variant::COLOR, "html", Variant::STRING); // TODO: These already have "from" constructors and I'm not sure where that code is. It doesn't show up in a grep for color and from.
```

Comment:
These are generated automatically for any legal variant conversion by https://github.com/godotengine/godot/blob/f46bf8e721d17f263437f4786454401b52721bc6/core/variant_call.cpp#L1384-L1399

If you want to change that, you'd have to remove the legal conversion between int/String and Color, and add a custom constructor instead.

Reply:
The main reason would be to have more descriptive argument names. I'm not an expert on Variant so I don't know if there are any big downsides to doing this.

Reply:
Sorry somehow didn't see the update two weeks ago. Super busy these days so I don't think I can spend the time to do this and test.